website/content/docs/internals/telemetry/metrics/authn.mdx

```diff
@@ -0,0 +1,95 @@
+---
+layout: docs
+page_title: "Telemetry reference: Authentication metrics"
+description: >-
+  Technical reference for authentication related telemetry values.
+---
+
+# Authentication telemetry
+
+Authentication telemetry provides information on authentication-related
```

Comment:
The pedant in me is a little uncomfortable lumping identity and expiration under "authentication", but they're certainly adjacent, so it's not entirely wrong.

Reply:
I'm open to suggestions if there's a better umbrella for this page. Alternatively, if folks are likely to look for them independently, we can break them into two different indexes. IMO, they're related enough that folks would be interested in both, so having them together on a page makes sense, but that's just an opinion.